packages/flutter_tools/lib/src/build_system/targets/web.dart

```diff
@@ -340,10 +340,6 @@ class WebReleaseBundle extends Target {
       if (environment.fileSystem.path.basename(inputFile.path) == 'index.html') {
```

Comment:
(The comment above still mentions that we're adding a hash for main.dart.js, but it seems we don't do that anymore, I think it should be updated just to not confuse future us :P)

Reply:
Fixed!